pkg/kn/commands/test_helper.go

```diff
@@ -38,6 +52,51 @@ func CreateTestKnCommand(cmd *cobra.Command, knParams *KnParams) (*cobra.Command
 	return knCommand, fakeServing, buf
 }
 
+// TestContains is a test helper function, checking if a substring is present in given
+// output string
+func TestContains(t *testing.T, output string, sub []string, element string) {
```

Comment:
I don't think you should start your func with "Test" as this is a trigger for unit tests functions (in _test.go files, though). Maybe just `ContainsAll` ? 

Ah, I see, you just have moved that functions (not used by yourself). So lets keep it for now and clean it up in another PR.

Reply:
Cool, agree. I don't think I added this func.